README.md

```diff
@@ -5,74 +5,152 @@
+  for each SRE team to isolate their access. For example, we can create a
+  “team1-ns” namespace for team1, and “team2-ns” namespace for team2, such that
+  both teams won’t be able to access each other’s services.
+- If Istio CA is compromised, all its managed key & cert in the cluster may be
+  exposed. We strongly recommend to run Istio CA on a dedicated namespace
+  (e.g., istio-ca-ns) which only cluster admins have access to.
   - We are looking into running Isito CA in the kubernetes master in the future releases.
-  
+
 ### Example
-Let’s consider a 3-tier application with three services: photo-frontend, photo-backend, and datastore. Photo-frontend and photo-backend services are managed by the photo SRE team while the datastore service is managed by the datastore SRE team. Photo-frontend can access photo-backend, and photo-backend can access datastore. However, photo-frontend cannot access datastore.
- 
-In this scenario, a cluster admin can creates 3 namespaces: istio-ca-ns, photo-ns, and datastore-ns. Admin has access to all namespaces, and each team only has access to its own namespace. The photo SRE team creates 2 service accounts to run photo-frontend and photo-backend respectively in namespace photo-ns. The datastore SRE team creates 1 service account to run the datastore service in namespace datastore-ns. Moreover, we need to enforce the service access control in [Istio Mixer](https://github.com/istio/mixer) such that photo-frontend cannot access datastore.
- 
-In this setup, Istio CA is able to provide key/cert management for all namespaces. And we successfully prevent the team from messing up services running by other teams.
+
+Let’s consider a 3-tier application with three services: photo-frontend,
+photo-backend, and datastore. Photo-frontend and photo-backend services are
+managed by the photo SRE team while the datastore service is managed by the
+datastore SRE team. Photo-frontend can access photo-backend, and photo-backend
+can access datastore. However, photo-frontend cannot access datastore.
+
+In this scenario, a cluster admin can creates 3 namespaces: istio-ca-ns,
+photo-ns, and datastore-ns. Admin has access to all namespaces, and each team
+only has access to its own namespace. The photo SRE team creates 2 service
+accounts to run photo-frontend and photo-backend respectively in namespace
+photo-ns. The datastore SRE team creates 1 service account to run the datastore
+service in namespace datastore-ns. Moreover, we need to enforce the service
+access control in [Istio Mixer](https://github.com/istio/mixer) such that
+photo-frontend cannot access datastore.
+
+In this setup, Istio CA is able to provide key/cert management for all
+namespaces. And we successfully prevent the team from messing up services
+running by other teams.
 
 ## Future Work
+
 - Fine-grained authorization and auditing
 - Secure Istio components (mixer, discovery service, etc.)
```

Comment:
Should we call this Istio Mixer and Istio Manager, to be consistent with the ones above?